api/v1/ytsaurus_types.go

```diff
@@ -72,6 +72,7 @@ type LocationType string
 const (
 	LocationTypeChunkStore       LocationType = "ChunkStore"
 	LocationTypeChunkCache       LocationType = "ChunkCache"
+	LocationTypeChunkLayer       LocationType = "ChunkLayer"
```

Comment:
It seems that `chunk` part is makes no sense here. Suggesting to call this field `LocationTypeLayers` or even `LocationTypePortoLayers` and change const value accordingly.